libs/designsystem/src/lib/components/avatar/avatar.component.scss

```
@@ -138,6 +138,7 @@ $badge-diameter: $avatar-badge-size;
       $main-colors,
       (
         'white': #ffffff,
+        'semi-light': get-color('semi-light'),
```

Shouldn't this be aligned?
```suggestion
        'white': get-color('white'),
        'semi-light': get-color('semi-light'),
```